### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by sindhus**
Add note about using Virtualenv and split installation methods into two sections.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,7 +26,13 @@ the right to make this assignment.
 
 -----
 
-Hasjob runs on [Python][] with the [Flask][] microframework. You will need
+Hasjob runs on [Python][] with the [Flask][] microframework. You can choose to setup your developmenet environment in the following two ways…
+
+### Virutalenv + Pip/easy_install
+
+If you are going to use a computer on which you would work on multiple Python based projects, it is recommended to use [Virtualenv](docs.python-guide.org/en/latest/dev/virtualenvs/) to keep isolated.
```
Comment (jace) [inline README.md]: I'd change this to:

Virtualenv is strongly recommended to ensure Hasjob's elaborate and sometimes version-specific requirements don't clash with anything else.

**Comment by sindhus** [comment]
No problem. Please feel free to reword any documentation related PR. Am just filling in the documentation that I think would help in getting up and running faster.

**Comment by jace** [comment]
If I accept this I have to do the grunt work of patching as well. :-p

**Comment by jace** [comment]
Bump.

**Comment by sindhus** [comment]
Don't have laptop with me today to do work. Will get back to this tomorrow?

**Comment by jace** [comment]
Sure. Will wait for it.

Kiran

-- 
Kiran Jonnalagadda
http://hasgeek.com/

(Sent from my phone)

**Pull request closed without merging**